src/utils/deployDiscourse.ts

```diff
@@ -5,80 +5,51 @@ import type { IProfile } from "../types/Profile";
 import createNetwork from "./createNetwork";
 import deploy from "./deploy";
 import rootFs from "./rootFs";
+import destroy from "./destroy";
+import checkVMExist from "./checkVM";
 
 const {
   generateString,
   MachinesModel,
   DiskModel,
-  GridClient,
   MachineModel,
   GatewayNameModel,
 } = window.configs?.grid3_client ?? {};
 
-const { HTTPMessageBusClient } = window.configs?.client ?? {};
-
-const DISCOURSE_FLIST =
-  "https://hub.grid.tf/rafybenjamin.3bot/threefolddev-discourse-v4.0.flist";
-
 export default async function deployDiscourse(
   data: Discourse,
   profile: IProfile
 ) {
-  const name = data.name;
-  const { mnemonics, storeSecret, networkEnv, sshKey } = profile;
-
-  const http = new HTTPMessageBusClient(0, "", "", "");
-  const client = new GridClient(
-    networkEnv as any,
-    mnemonics,
-    storeSecret,
-    http,
-    undefined,
-    "tfkvstore" as any
+  let domainName = await getUniqueDomainName(
+    profile,
+    data.name,
+    "discourse",
+    "dc"
   );
 
-  await client.connect();
-
-  let randomSuffix = generateString(10).toLowerCase();
-
-  const network = createNetwork(new Network(`nw${randomSuffix}`, "10.200.0.0/16")); // prettier-ignore
-
-  let domainName = await getUniqueDomainName(client, "dc", name);
-
   let [publicNodeId, nodeDomain] = await selectGatewayNode();
-  const domain = `${domainName}.${nodeDomain}`;
+  data.domain = `${domainName}.${nodeDomain}`;
 
-  await depoloyDiscourseVM(data, profile, domain, network, randomSuffix);
+  const deploymentInfo = await depoloyDiscourseVM(data, profile);
 
-  const discourseInfo = await getDiscourseInfo(client, name);
-  const planetaryIP = discourseInfo[0]["planetary"] as string;
-  const publicIP = discourseInfo[0]["publicIP"]
-    ? discourseInfo[0]["publicIP"]["ip"].split("/")[0]
-    : "";
+  const planetaryIP = deploymentInfo["planetary"] as string;
+
+  // const publicIP = deploymentInfo[0]["publicIP"]
```

Comment:
forgot to uncomment ?

Reply:
currently, we do not use the publicIP anywhere. comment it to be there if it needed later